### ISSUE TITLE
Would be nice to see numerical temperature indicator for the bed and extruder temperature

### ISSUE THREAD

**Issue by msurguy** [daid/LegacyCura#324]
Using 12.12, the current graph doesn't do justice of displaying the temperature, would like to see  actual realtime number anywhere on the chart. Would anyone like me to implement this feature?

**Comment by dwalton64** [comment]
I would love to see that feature.  I looked at the code to see how to implement it (gui\printWindow.py) but I can't tell how UpdateDrawing() gets passed the temperature targets for the heater and the bed heater.  I think the variables are tempSP and bedTempSP?  I have a heated bed that heats very slowly and end up setting the extruder heater to 70 while I wait for the bed to heat so I can tell how close the bed is to 70.  The bed heater has a target line in the code, but the target value for the bed does not appear to be set and is not plotted.  

In any case, I would be happy to help implement the feature, but can't tell how the target temps are passed into UpdateDrawing().  It might be cleaner to pass them as arguments.

**Comment by daid** [comment]
the SP variables are the SetPoints, so the target values. The values are passed in the self.points array

**Comment by dwalton64** [comment]
Thanks Daid.  I figured that out and added a couple of lines to PrintWindow.py in the UpdateDrawing method:

``` python
            t0 = temp
            bt0 = bedTemp
            tSP0 = tempSP
            btSP0 = bedTempSP
            x0 = x1 + 1

        tempString = 'Target Temp: {0}   Bed Target Temp: {1}'
        dc.DrawText(tempString.format(tempSP, bedTempSP), 5, 5)
        del dc
        self.Refresh(eraseBackground=False)
        self.Update()
```

The tempstring and DrawText lines are the lines I added.  It wasn't real pretty, but it proved to me that the code thought bedTempSP was equal to 0.  I have a heated bed and the target temp was actually 70.

I see that bedTempSP gets set from self.points.  It looks like  AddPoint() is called from mcTempUpdate().  Is that a call back from machineCom.py?  I was a little confused by MachineComPrintCallback implementing a method also called mcTempUpdate (that method is simply a pass statement.)

I haven't been able to figure out why the bedTempSP is not getting set.

Thanks for an awesome tool! I appreciate you helping us.
